sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -53,47 +60,253 @@ def _is_ipython(self):
             if ipy is None:
                 return False
         except ImportError:
             return False
         return True
 
-    def create_experiment(self, exp_object, namespace=None):
+    def create_experiment(
+        self, exp_object, namespace=utils.get_default_target_namespace()
+    ):
         """Create the Katib Experiment.
 
         :param exp_object: Experiment object.
         :param namespace: Experiment namespace.
         If the namespace is None, it takes namespace from the Experiment or "default".
-
-        :return: Created Experiment.
-        :rtype: dict
         """
 
-        if namespace is None:
-            namespace = utils.set_katib_namespace(exp_object)
         try:
-            outputs = self.api_instance.create_namespaced_custom_object(
+            self.api_instance.create_namespaced_custom_object(
                 constants.KUBEFLOW_GROUP,
                 constants.KATIB_VERSION,
                 namespace,
                 constants.EXPERIMENT_PLURAL,
-                exp_object)
+                exp_object,
+            )
         except client.rest.ApiException as e:
             raise RuntimeError(
                 "Exception when calling CustomObjectsApi->create_namespaced_custom_object:\
-         %s\n" % e)
+         %s\n"
+                % e
+            )
+
+        # TODO (andreyvelich): Use proper logger.
+        print("Experiment {} has been created".format(exp_object.metadata.name))
 
         if self._is_ipython():
             if self.in_cluster:
                 import IPython
-                IPython.display.display(IPython.display.HTML(
-                    'Katib Experiment {} '
-                    'link <a href="/_/katib/#/katib/hp_monitor/{}/{}" target="_blank">here</a>'.format(
-                        exp_object.metadata.name, namespace, exp_object.metadata.name)
-                ))
-        return outputs
+
+                IPython.display.display(
+                    IPython.display.HTML(
+                        "Katib Experiment {} "
+                        'link <a href="/_/katib/#/katib/hp_monitor/{}/{}" target="_blank">here</a>'.format(
+                            exp_object.metadata.name,
+                            namespace,
+                            exp_object.metadata.name,
+                        )
+                    )
+                )
+
+    def tune(
+        self,
+        # TODO (andreyvelich): How to be consistent with other APIs (name) ?
+        name: str,
+        objective: Callable,
+        parameters: Dict[str, Any],
+        base_image: str = constants.BASE_IMAGE_TENSORFLOW,
+        namespace: str = utils.get_default_target_namespace(),
+        algorithm_name: str = "random",
+        objective_metric_name: str = None,
+        additional_metric_names: List[str] = [],
+        objective_type: str = "maximize",
+        objective_goal: float = None,
+        max_trial_count: int = None,
+        parallel_trial_count: int = None,
```

Comment:
how about these fields' default values to `1` for who are not familiar with katib experiments?

Reply:
We set default Trial Threshold here: https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1beta1/experiment_defaults.go#L35-L39
I am not sure, if we want to introduce another default value in the `tune` API.
WDYT @anencore94 @tenzen-y ?

Reply:
I introduce this because of the same reason with this comment: https://github.com/kubeflow/katib/pull/1951#discussion_r972651507.

I'm not sure if we have to implement such fast-validation (which is duplicated) or not.

Reply:
Since introducing duplicated values will cause the bug, it would be good not to set the default values in this.

Reply:
I agree with you, but then I think we should write docstring and tutorial page more detailed and kind!

Reply:
@anencore94 @tenzen-y Maybe we update docstring with link to [this page](https://www.kubeflow.org/docs/components/katib/experiment/#configuration-spec) for Trial budget info?

Reply:
That sounds good.